README.md

```diff
@@ -1,11 +1,11 @@
-![360Giving Logo](http://openphilanthropy.files.wordpress.com/2014/02/cropped-360givinglogo-2010-size.jpg)
+![360Giving Logo](./360standard-color.png)
 
 360Giving Standard
 ========
 
 360Giving aims to help UK grant makers and philanthropists to publish their grant information online in an easy to use way.  We are motivated by a commitment to the value of open data and strategic funding.
```

Comment:
update text to align with current mission statement: 

360Giving's mission is to help UK funders publish open, standardised grants data, and empower people to use it to improve charitable giving